examples/react/w3console/src/share.tsx

```diff
@@ -22,15 +25,26 @@ export async function toCarBlob (delegation: Delegation): Promise<Blob> {
     carParts.push(chunk)
   }
   const car = new Blob(carParts, {
-    type: 'application/vnd.ipld.car'
+    type: 'application/vnd.ipfs.car'
   })
   return car
 }
 
-export function SpaceShare (): JSX.Element {
-  const [, { createDelegation }] = useKeyring()
+export async function toDelegation (car: Blob): Promise<Delegation> {
```

Comment:
is there anything we should do here to ensure this is actually a delegation or is that handled in the library?

Reply:
yes, the code should catch where the blob cannot be decoded as a CAR and let the user know that the file isn't a CAR (or corrupted). 

and it should catch an error from importDAG if the CAR doesn't contain blocks that can be parsed as a ucan delegation.

This feature in general is very much a proof of concept, so i'd prefer to land it as is, and then prioritise adding the good feedback if we decide it's a thing we want to keep. It'll be much less relevant once we can fetch users other spaces for them from accounts, and share spaces by email address.